documentation/src/main/asciidoc/introduction/Entities.adoc

```diff
@@ -1233,7 +1233,7 @@ class Book {
 
 Remember, if we wish to the modify the collection we must <<bidirectional-problem,change the owning side>>.
 
-We've again used ``Set``s to represent the association.
+We've again used `Set` to represent the association.
```

Comment:
Asciidoc requires double-backticks when there are no spaces around the literal. So double backticks here are necessary.

Reply:
But it's pluralized because we used `Set` twice, once on each side of the association, in the preceding code examples.

I don't see anything wrong here, and I don't think the change is an improvement.

Reply:
I reverted back.